virtru/virtuoso-design-system
# fix: correct danger button styles [PLAT-675]

changes: _rev_

### Proposed Changes
fix: correct danger button styles [PLAT-675]

- PLAT-675
  - correct danger button styles

### Checklist

- [x] I have added or updated documentation/readme (if appropriate)

### Testing Instructions

1. Use storybook deployment for the master branch of `virtuoso-design-system`
2. Check Danger button behaves and has corresponding colors. 

[PLAT-675]: https://virtru.atlassian.net/browse/PLAT-675
